apis/v1alpha1/certificate.go

```diff
@@ -22,7 +22,8 @@ import (
 
 // CertificateSpec defines the desired state of Certificate.
 type CertificateSpec struct {
-
+	Certificate    *ackv1alpha1.SecretKeyReference `json:"certificate,omitempty"`
+	CertificateARN *string                         `json:"certificateARN,omitempty"`
```

Comment:
We might want to add some custom documentation for these fields. I'm getting a bit confused with all the Certificate* fields 😄

Reply:
Good point. I'll add it.

Reply:
Done.